lib/ops/opsservice/shrink.go

```diff
@@ -454,62 +455,67 @@ func (s *site) waitForServerToDisappear(hostname string) error {
 	return trace.Wrap(err)
 }
 
-func (s *site) removeFromEtcd(ctx *operationContext, runner *serverRunner, server storage.Server) error {
-	out, err := runner.Run(s.etcdctlCommand("member", "list")...)
-	if err != nil {
-		return trace.Wrap(err, "failed to list etcd members: %s", out)
-	}
-
-	provisionedServer := ProvisionedServer{Server: server}
-	memberID, err := utils.FindETCDMemberID(
-		string(out), provisionedServer.EtcdMemberName(s.domainName))
-	if err != nil {
+func (s *site) removeFromEtcd(ctx context.Context, opCtx *operationContext, server storage.Server) error {
+	peerURL := server.EtcdPeerURL()
+	logger := opCtx.WithField("peer", peerURL)
+	logger.Info("Remove peer from etcd cluster.")
+	b := utils.NewExponentialBackOff(defaults.EtcdRemoveMemberTimeout)
```

Comment:
Does this exponential backoff has reasonable max retry interval? To make sure we get enough retry attempts.

Reply:
I'll fix the max interval at 10s